scan-local.sh

```diff
@@ -22,11 +22,30 @@ docker run -d --name "$name" --volumes-from "$name-data" repo-info:local > /dev/
 
 echo '# `'"$image"'`'
 
+size="$(
+	docker inspect -f '{{ .VirtualSize }}' "$image" | awk '{
+		oneKb = 1000;
+		oneMb = 1000 * oneKb;
+		oneGb = 1000 * oneMb;
+		if ($1 >= oneGb) {
+			printf "~ %.2f Gb", $1 / oneGb
+		} else if ($1 >= oneMb) {
+			printf "~ %.2f Mb", $1 / oneMb
+		} else if ($1 >= oneKb) {
+			printf "~ %.2f Kb", $1 / oneKb
```

Comment:
(I started with it being 1024-based, and found I was getting the "wrong" answers relative to what `docker images` showed, which I think is what our users will be expecting.)